changelog.txt

```diff
@@ -1,3 +1,18 @@
+4.2.0
+=============
+New Features:
+- **Allow users to specify their own instance metadata**.  For details see https://aka.ms/msal-net-custom-instance-metadata [More information on this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1259)
```

Comment:
For some of these it may be useful to link to the wiki page explaining about the functionality instead of having people to go to the issue to find that? Perhaps a link to the blog post section for each of the most notable changes?